desktop/webpack.config.base.js

```diff
@@ -15,8 +15,9 @@ module.exports = {
       loader: 'babel',
       exclude: /(node_modules|\/dist\/)/,
       query: {
-        optional: ['runtime'],
-        stage: 2
+        cacheDirectory: true,
```

Comment:
I don't think we really need to. the filename the cache uses is based on a sha1 of the file contents and the options used so i think its pretty safe:
https://github.com/babel/babel-loader/blob/2fa087a8a48407d6e0b161de8254a92bfae260a8/lib/fs-cache.js#L130

Safe enough they're talking about having it be on by default:
https://github.com/babel/babel-loader/issues/199

If we run into problems we can turn it off